src/MSALWrapper/AuthFlow/IntegratedWindowsAuthentication.cs

```diff
@@ -72,7 +72,8 @@ protected override async Task<TokenResult> GetTokenInnerAsync()
             }
             catch (MsalClientException ex) when (ex.Message.Contains("WS-Trust endpoint not found"))
             {
-                this.logger.LogWarning($"IWA only works on Corp Net, please turn on VPN.");
+                this.logger.LogWarning($"IWA only works on corporate AD backed network, AzureAuth is trying to use other auth flows.");

```

Comment:
I think the problem here is that this is logged as Warning, and it generally doesn't need to be a warning given, most users will either succeed here, or in web mode. My recommendation is to change the verbosity level to debug, so it's till present in the logs, but not shown on the CLI during normal usage. The text itself, could be left alone.